nixos/modules/services/web-servers/ttyd.nix

```diff
@@ -64,8 +64,7 @@ in
         apply = value: if value == null then null else toString value;
         description = lib.mdDoc ''
           File containing the password to use for basic authentication.
-          For insecurely putting the password in the globally readable store use
-          `pkgs.writeText "ttydpw" "MyPassword"`.
+          For setting the password use `pkgs.writeText "ttydpw" "MyPassword"`.
```

Comment:
This is still insecure right? Hiding the store path is by no means a security mechanism.

Reply:
it totally is. Sorry not sure what I thought here.

Maybe we just keep the test and the rest gets removed?

Reply:
The use of `LoadCredential` might also be kept, only the `description` changes shall be reverted.